traceevents/src/main/java/com/tomtom/kotlin/traceevents/Tracer.kt

```diff
@@ -166,8 +166,20 @@ public class Tracer private constructor(
     private val taggingClassName: String,
     private val context: String
 ) : InvocationHandler {
+    /**
+     * A data class used to store the annotations of a trace event method.
+     * For performance reasons, this data is only retrieved once per method and then stored in the cache.
+     */
+    private data class MethodAnnotationsCache(
```

Comment:
From the Kotlin code style:

> Put nested classes next to the code that uses those classes.

So I would put this class after `methodAnnotationsCache`.